README.md

```diff
@@ -1,4 +1,14 @@
 # aws-beanstalk-deploy
 Automated deploy for AWS ElasticBeanstalk through bitbucket pipelines.
 
-This script was written to work with release branches following a default style: `release-v0.0.0.0-prod` and produces a deploy version: `v0.0.0.0-prod`
+## Requirements
+To use that script you will need to use a bitbucket pipeline.
+We use release branchs to deploy our applications following a default style: `release-v0.0.0.0-prod` and producing `v0.0.0.0-prod` as deployed version.
```

Comment:
Aqui é bom especificar que no caso de release é branch e o deploy é feito para staging e o outro caso é para produção com tag. Tirar um zero tbm:

release branch for staging `release-v0.0.0-prod` and tag `v0.0.0-prod` for production.